src/main/java/com/knu/buga1chuk/util/Base64Utils.java

```diff
@@ -0,0 +1,19 @@
+package com.knu.buga1chuk.util;
+
+import java.util.Base64;
+
+public class Base64Utils {
```

Comment:
It is an utility class.
And something is missing...

Reply:
final
Fix